docs/apache-airflow/core-concepts/params.rst

```diff
@@ -159,6 +159,11 @@ JSON Schema Validation
         },
     ):
 
+.. note::
+    Schema validation on DAG definitions is only made if a schedule is defined. If a manual or external trigger
```

Comment:
```suggestion
    Schema validation on DAG definitions is only made if ``schedule`` is provided. If a manual or external trigger
```
only to avoid confusion with notion of defined vs undefined